internal/handlers/common/aggregations/stages/projection/projection.go

```diff
@@ -312,6 +345,21 @@ func projectDocumentWithoutID(doc *types.Document, projection *types.Document, i
 			var op operators.Operator
 			var v any
 
+			if !operators.IsOperator(value) {
+				if value.Len() == 0 {
+					return nil, commonerrors.NewCommandErrorMsgWithArgument(
+						commonerrors.ErrEmptySubProject,
+						"Invalid $project :: caused by :: An empty sub-projection is not a valid value."+
+							" Found empty object at path",
+						"$project (stage)",
+					)
+				}
```

Comment:
Also great detecting this. Codecov thinks it's missing test for it. Could you add it too please :pray:

Reply:
The same thing as above, we have a test for this but it's catched on validation

Reply:
After DS we decided to remove both unreachable code as everything is already catched on validation

Reply:
Good idea of removing unreachable code :+1: